new_experiment/find_specific_genes_in_new_experiment.py

```diff
@@ -55,6 +55,8 @@
 # The remaining parameters specify filenames that intermediate data files will be written to.
 #
 # The user also needs to provide metadata files:
+#
+# These files should be located in `data/metadata/` directory.
```

Comment:
You may also want to add an empty file to `data/metadata` called `PLACE_GROUPS_FILE_HERE` or something

Reply:
Good idea!